qrest/conf.py

```diff
@@ -41,6 +42,8 @@ def __init__(
         default: Optional[str] = None,
         choices: Optional[list] = None,
         description: Optional[str] = None,
+        schema: Optional[str] = None,
+        example: Optional = None
```

Comment:
You cannot have just `Optional` here, if I'm not mistaken you need a type. Because it can be anything, I propose `Optional[Any]`.